projects/Mallard/src/navigation/navigators/underlay/transition.tsx

```diff
@@ -13,7 +13,7 @@ const issueScreenToIssueList = (sceneProps: NavigationTransitionProps) => {
     const { height: windowHeight, width } = Dimensions.get('window')
     const isTablet = width >= Breakpoints.tabletVertical
 
-    const finalTranslate = windowHeight - 80
+    const finalTranslate = windowHeight - windowHeight / 4
```

Comment:
Could this cause issues on some weird squat Android devices (maybe even an iPhone 5). Do we know that the user will be able to select every issue?

Reply:
Will try an iphone5.

Reply:
You're right - looks poor on 5S.